packages/styles/scss/components/_room.scss

```
@@ -6,7 +6,8 @@
   width: 100%;
   height: 100%;
 
-  * {
+  *[class^='lk-'],
+  *[class*=' lk-'] {
```

If we're using the `attr*=''` selector here, I think we could get rid of the prefixed attribute selector and simplify this a bit. If an element has a class containing `lk-`, that should always match our conditions I think?

```suggestion
  *[class*='lk-'] {
```

---

having both in there matches for
```html
<!-- match -->
<div class="lk-something" />
<!-- match -->
<div class="my-class lk-something" />
<!-- no match -->
<div class="bulk-something" />
```

the last example with "bulk-something" would still match if we did _only_  `*[class*='lk-']`

---

but yeah, only using this pattern as a quick fix. If there's a cleaner solution you have in mind, feel free to change it up and kick this out entirely!